lib/cc/engine/analyzers/file_list.rb

```diff
@@ -0,0 +1,33 @@
+module CC
+  module Engine
+    module Analyzers
+      class FileList
+        def initialize(directory:, engine_config:, extension:)
+          @directory = directory
+          @engine_config = engine_config
+          @extension = extension
+        end
+
+        def files
+          files = Dir.glob("#{directory}/**/*.#{extension}").reject do |f|
+            File.directory?(f)
+          end
+          files - excluded_files
```

Comment:
That's fair. Could we at least either rename the method or the variable so that they're not the same though?